bin/register.js

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env node
+"use strict";
+const shell = require('shelljs');
+const path = require('path');
+shell.exec(path.normalize(__dirname + "/../systemd_register.sh"));
```

Comment:
Proc se vola register a unregister pres javascript?

Reply:
aby to šlo používat jako CLI